pkg/util/kubernetes/kubelet/podwatcher.go

```diff
@@ -116,7 +112,8 @@ func (w *PodWatcher) computeChanges(podList []*Pod) ([]*Pod, error) {
 			w.oldReadiness[container.ID] = isPodReady
 
 			if isPodReady {
-				w.lastSeenReady[container.ID] = now
+				pod.UpdateLastSeenReady(idx, now)
+				updatedContainer = true
```

Comment:
I feel like we will generate lots of Pod update in workload meta due to the fact that the LastSeenReady will be updated all the time.

Have you considered introducing a `ReadySince` field instead?